Create a bug report and feature request template (#728)
single-cell-data/TileDB-SOMA: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Suggest an idea for this project
title: "[FEATURE REQUEST]"
labels: ''
assignees: ''

---

**Is your feature request related to a problem? Please describe.**

> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Reviewer: ```suggestion
A clear and concise description of what the problem is. Ex. It would be amazing if I could [...]
```
To be more positive... :-)
